packages/plugin-ext/src/main/browser/debug/plugin-debug-service.ts

```diff
@@ -92,6 +78,16 @@ export class PluginDebugService implements DebugService, PluginDebugAdapterContr
         this.contributors.delete(debugType);
     }
 
+    registerDebugConfigurationProvider(provider: PluginDebugConfigurationProvider): Disposable {
+        const handle = provider.handle;
+        this.configurationProviders.set(handle, provider);
+        return Disposable.create(() => this.unRegisterDebugConfigurationProvider(handle));
+    }
+
+    unRegisterDebugConfigurationProvider(handle: number): void {
```

Comment:
It is more common to write `unregister` as a single word, rather than `unRegister`:

```suggestion
    unregisterDebugConfigurationProvider(handle: number): void {
```